src/main/java/com/minecolonies/coremod/entity/ai/mobs/EntityAIBarbarianAttackMelee.java

```diff
@@ -0,0 +1,98 @@
+package com.minecolonies.coremod.entity.ai.mobs;
+
+import com.minecolonies.api.util.ItemStackUtils;
+import net.minecraft.entity.EntityCreature;
+import net.minecraft.entity.EntityLivingBase;
+import net.minecraft.entity.ai.EntityAIBase;
+import net.minecraft.init.SoundEvents;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.ItemSword;
+import net.minecraft.util.DamageSource;
+import net.minecraft.util.EnumHand;
+
+/**
+ * Created by Asher on 13/6/17.
+ */
+public class EntityAIBarbarianAttackMelee extends EntityAIBase
+{
+
+    final private EntityCreature   entity;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AModifiersOrderCheck)